extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -58,11 +55,12 @@ interface RepoListQuickPickItem extends QuickPickItem {
  * Gets the repositories to run the query against.
  */
 async function getRepositories(): Promise<string[] | undefined> {
-  if (REPO_LISTS) {
-    const quickPickItems = Object.keys(REPO_LISTS).map<RepoListQuickPickItem>(key => (
+  const REPO_LISTS = getRemoteRepositoryLists();
```

Comment:
nit: Typically, if something is not a module-level constant, then we avoid using CAPs for it.

```suggestion
  const repoLists = getRemoteRepositoryLists();
```